Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -41,22 +45,24 @@ struct NoImageModeDefaults {
     static let ScriptName = "images"
 }
 
-
 enum BlockingStrength: String {
     case basic
     case strict
 
     static let allOptions: [BlockingStrength] = [.basic, .strict]
 }
 
-
 @available(iOS 11.0, *)
 class ContentBlockerHelper {
-    var stats = TPPageStats()
+    var stats = TPPageStats() {
+        didSet {
+            NotificationCenter.default.post(name: .TPPageStatsChanged, object: tab)
```

Comment:
stats change pretty frequently. Can we check the old stats and only alert when stats go from 0 to >1 and >1 to 0?